packages/client/test/integration/StreamEndpoints.test.ts

```diff
@@ -83,8 +83,43 @@ function TestStreamEndpoints(getName: () => string, delay: number) {
             expect(newStream.id).toEqual(expectedId)
         })
 
-        it('invalid id', async () => {
-            await expect(async () => client.createStream({ id: 'invalid.eth/foobar' })).rejects.toThrow()
+        it('legacy format', async () => {
+            const streamId = '7wa7APtlTq6EC5iTCBy6dw'
+            await expect(async () => client.createStream({ id: streamId })).rejects.toThrow(`stream id "${streamId}" not valid`)
+        })
+
+        it('key-exchange format', async () => {
+            const streamId = 'SYSTEM/keyexchange/0x0000000000000000000000000000000000000000'
+            await expect(async () => client.createStream({ id: streamId })).rejects.toThrow(`stream id "${streamId}" not valid`)
+        })
+
+        describe('ENS', () => {
+
+            it('domain owned by user', async () => {
+                const streamId = 'testdomain1.eth/foobar'
```

Comment:
Will this test fail if run multiple times due to `testdomain1.eth/foobar` already being present in docker smart contract? If so maybe some random `uuid()` needs to be added as part of the path.

Reply:
Good catch, fixed